src/codemagic/utilities/argument_utils.py

```diff
@@ -0,0 +1,8 @@
+def get_binary_arguments_value(value, not_value):
+    if value and not_value:
+        raise Exception
```

Comment:
Too broad exception. `ValueError` with appropriate message would suit a lot more.

Reply:
will just delete it